kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/NMDbusConnectorTest.java

```diff
@@ -1184,6 +1258,14 @@ private void thenNetInterfaceTypeIs(NetworkInterfaceType type) {
     public void thenConfigurationEnforcementIsActive(boolean expectedValue) {
         assertEquals(expectedValue, this.instanceNMDbusConnector.configurationEnforcementIsActive());
     }
+    
+    private void thenConnectionIsDeleted(String path) {
+        verify(mockedConnections.get(path)).Delete();
```

Comment:
Be specific. You expect only *one* call. If there's more than one there's something wrong.

```suggestion
        verify(mockedConnections.get(path), times(1)).Delete();
```